### PR TITLE
Give character command

### PR DESCRIPTION
**Pull request by SpikeHD**
A command for adding characters to a players roster.

### CODE REVIEW

**Comment by SpikeHD** [comment]
Not done yet gimme a sec to bring it up-to-speed with the conventions for everything else

**Comment by SpikeHD** [comment]
Alright, tested and ready :)

File: src/main/java/emu/grasscutter/commands/PlayerCommands.java [hunk 1]
```diff
@@ -176,6 +177,110 @@ public void execute(GenshinPlayer player, List<String> args) {
...
         }
     }
+
+    @Command(label = "givechar", aliases = { "givec" }, usage = "Usage: givechar <playerId> <avatarId> [level]")
+    public static class GiveCharCommand implements CommandHandler {
+        @Override public void execute(GenshinPlayer player, List<String> args) {
+            int target, avatarID, level = 1;
+
+            if(args.size() < 2) {
+                CommandHandler.sendMessage(null, "Usage: give <player> <avatarId> [level]");
```
Comment (MlgmXyysd) [inline src/main/java/emu/grasscutter/commands/PlayerCommands.java:187]: `give` instead of `givechar`?

Comment (4Benj) [reply src/main/java/emu/grasscutter/commands/PlayerCommands.java:187]: Maybe make the player variable optional when the player is submitting the command

Comment (SpikeHD) [reply src/main/java/emu/grasscutter/commands/PlayerCommands.java]: > `give` instead of `givechar`?

Good catch, that's fixed!

Comment (MlgmXyysd) [reply src/main/java/emu/grasscutter/commands/PlayerCommands.java]: > Maybe make the player variable optional when the player is submitting the command

makes sense

**Comment by Melledy** [comment]
When directly setting levels on avatars, you need to also take in account their ascension level and ascension talents.

**Comment by SpikeHD** [comment]
> ascension talents

Good call, I will make sure that's added

**Comment by SpikeHD** [comment]
Both ascension levels and talents should be properly given and calculated now

**Pull request merged**